src/zimcheck.cpp

```diff
@@ -523,7 +525,7 @@ void test_articles(const zim::File& f, ErrorLogger& reporter, ProgressBar progre
                     if (s1 != s2 )
                         continue;
 
-                    reporter.setTestResult(TestType::REDUNDANT, false);
+                    reporter.setTestResult(TestType::REDUNDANT, true);
```

Comment:
We must not set the test result as ok.
We must set it to failling and make the return code return 1 only if mandatory (error) test are failing (and keep returning 0 if there is only warning).

Reply:
Agree.